onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -446,6 +447,17 @@ def _get_dynamic_input_quantization_params_uint8(self, input_name, nodes_list):
         input_scale_name = input_name + "_scale"
         input_zp_name = input_name + "_zero_point"
 
+        # Add tensors for quantize range and zero value.
```

Comment:
Description (maybe a formula?) about the behavior of this subgraph would be nice.